linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml

```diff
@@ -0,0 +1,18 @@
+# platform = Red Hat Enterprise Linux 7,multi_platform_fedora
+# reboot = true
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Verify that Interactive Boot is Disabled in /etc/default/grub
+  replace:
+    dest: /etc/default/grub
+    regexp: systemd.confirm_spawn=(1|yes|true|on)
+    replace: systemd.confirm_spawn=no
+  tags:
+    @ANSIBLE_TAGS@
+
+- name: Verify that Interactive Boot is Disabled (runtime)
+  shell: /sbin/grubby --update-kernel=ALL --remove-args="systemd.confirm_spawn"
```

Comment:
@matusmarhefka can you confirm if grubby modifies `/etc/default/grub`?

Reply:
@redhatrises I believe it does not care about that file at all. https://bugzilla.redhat.com/show_bug.cgi?id=768106#c6

Reply:
Thanks @dahaic